velox/functions/lib/aggregates/CMakeLists.txt

```diff
@@ -18,6 +18,9 @@ add_library(velox_functions_aggregates SingleValueAccumulator.cpp
 target_link_libraries(velox_functions_aggregates velox_exec
                       velox_presto_serializer Folly::folly)
 
-if(${VELOX_BUILD_TESTING})
-  add_subdirectory(tests)
+# Currently there is no test under tests folder, so it looks a little strange
+# here, when there is tests under tests directory we need to rewrite this if
+# condition.
+if(${VELOX_BUILD_TESTING} OR ${VELOX_BUILD_TEST_UTILS})
```

Comment:
Can it be like this?
```
if(${VELOX_BUILD_TESTING})
  add_subdirectory(tests)
elseif(${VELOX_BUILD_TEST_UTILS})
  add_subdirectory(tests/utils)
endif()
```

Reply:
@xiaodouchen Since there is no test under `tests` directory now, I's prefer to keep the `if` simple for now, when we add tests into this directory, we have to change the CMakeLists file anyway(which is just several lines of code).

Reply:
+1

We can follow the methods in the CMakeLists.txt file in other directories, such as:
https://github.com/facebookincubator/velox/blob/f558c2e895a00bdcfa006dfb96f40ccd2f3630fd/velox/exec/CMakeLists.txt#L99-L103

or
https://github.com/facebookincubator/velox/blob/f558c2e895a00bdcfa006dfb96f40ccd2f3630fd/velox/functions/prestosql/CMakeLists.txt#L86-L91

Because there is currently no unit test in the "tests" dir, we can **leave an empty CMakeLists.txt file** in the "tests" directory and write only one sentence in it(also same as other directories): 
```
add_subdirectory(utils)
```

In this way, **all directories will be written in the same way**, and **the above comment section does not need any more**, because there is no need to modify it even if a unit test is added later.  

@xumingming  What do you think?

Reply:
@lingbin I'm ok if you insist.

Reply:
done.